Questão 1: Considerando o programa abaixo e sabendo que o usuário digitou 3 e 6 (nessa ordem), preencha no campo resposta o valor final da variável total:

In [1]:
num1 = int(input("Digite um número"))
num2 = int(input("Digite outro número"))

total = num2 + num1/2

print("O novo número é ", total)

Digite um número3
Digite outro número6
O novo número é  7.5


Questão 2: Considerando o programa abaixo e sabendo que o usuário digitou 200 e 40 (nessa ordem), o que seria impresso na ordem exata de execução?

In [2]:
salario = float(input("Digite o valor do salário"))
aumento = float(input("Digite o percentual de aumento"))

novoSalario = salario*(1+aumento/100)

print(novoSalario)
print(salario)

Digite o valor do salário300
Digite o percentual de aumento20
360.0
300.0


Questão 3: O programa abaixo foi executado diversas vezes e, após o usuário digitar um valor, o programa imprimiu a situação do aluno. Para cada valor digitado, coloque do lado a situação do aluno (“Reprovado”, “Recuperação” ou “Aprovado”):

In [7]:
media = float(input("Qual a média do aluno?"))

if 0 < media <=5:
    print("Reprovado")
else:
    if 5 < media < 6:
        print("Recuperação")
    else:
        print("Aprovado")

Qual a média do aluno?5
Reprovado


| Valor Digitado | Situação |
| --- |--- |
| 1 | Reprovado |
| 9| Aprovado |
| -2 | Aprovado |
| 6 | Aprovado |
| 5 | Reprovado |

Questão 4: Para cada execução do programa abaixo, indique na tabela os valores finais das variáveis precoA e precoB:

In [13]:
precoA = int(input("Digite o valor do produto 1"))
precoB = int(input("Digite o valor do produto 2"))

if precoA>precoB:
    precoB = precoB*2
else:
    if precoA==precoB:
        precoB = precoB*1.5
    else:
        precoA = precoB*2 + 10

print(precoA)
print(precoB)

Digite o valor do produto 180
Digite o valor do produto 290
190
90


| Valor Digitado 1 | Valor Digitado 2 | precoA | precoB
| --- |--- | --- |--- |
| 150 | 300 | 610 | 300 |
| 10 | 30 | 70 | 30 |
| 20 | 5 | 20 | 10  |
| 40 | 40 | 40 | 60  |
| 80 | 90 | 190 | 90 |

Questão 5: Ao executar o programa abaixo, qual é a média final do aluno?

In [14]:
notas = (7,4,3,8)
soma = 10
for nota in notas:
    soma = soma + nota
print("Média", soma/4)

Média 8.0


Questão 6: Ao executar o programa abaixo, o usuário digitou DF e SP (nessa ordem). Qual será o valor final da variável percentual?

In [15]:
origem = input("Digite um estado")
destino = input("Digite outro estado")

percentual = 18

if origem=="SP" and destino=="DF":
    percentual = 12
if origem=="GO" or destino=="DF":
    percentual = 7
else:
    percentual = 5
    
print("Percentual", percentual)

Digite um estadoDF
Digite outro estadoSP
Percentual 5


Questão 7: Um usuário pretendia digitar os números 3, 4, 1,2, 2, -2, 1, 2, 3, 1, 2, 0, mas o programa encerrou antes que ele digitasse todos os valores. Qual foi o último valor digitado?

In [17]:
soma = 0
rodadas = 10
num = 0
while soma <= rodadas:
    num = int(input("Digite um número de 1 a 10"))
    soma = soma + num
    if soma > 20:
        break
print(soma/4)

Digite um número de 1 a 103
Digite um número de 1 a 104
Digite um número de 1 a 101
Digite um número de 1 a 102
Digite um número de 1 a 102
3.0


Questão 8: Ao executar o programa abaixo, qual será o valor final da variável mediaFinal:

In [18]:
notas = (8,5,7,3)
mediaFinal = 0
soma = 0

for i in notas:
    if i < 4:
        mediaFinal = 0
    else:
        soma = soma + i

mediaFinal = soma / 4
print(mediaFinal)

5.0


Questão 9: Para evitar cálculos manuais, um cliente lhe solicitou que implementasse um programa de cálculo de reajuste de preços de 8 produtos da sua empresa mas lhe pediu que esse programa possa ser usado depois para cálculo de reajuste de outras listas de produtos. Continue a implementação abaixo desse programa considerando os requisitos acima, sabendo que você só precisa implementar de 2 a 4 linhas para o cálculo e uma para imprimir os novos valores de cada produto.

In [19]:
precos = (100,60,20,30,5,20)
reajuste = 1.20

for preco in precos:
    print("O novo preço é", preco*reajuste)

O novo preço é 120.0
O novo preço é 72.0
O novo preço é 24.0
O novo preço é 36.0
O novo preço é 6.0
O novo preço é 24.0
